vulcan/_client.py

```diff
@@ -50,15 +50,18 @@ def set_logging_level(logging_level: int):
         """
         log.setLevel(logging_level)
 
-    async def get_students(self, cached=True) -> List[Student]:
+    async def get_students(
+        self, state: StudentState = StudentState.ACTIVE, cached=True
```

Comment:
One last thing. You should add `state` **after** the `cached` argument. Because if you replace them like this, this method won't be backwards compatible.